packages/babel-parser/src/util/location.js

```diff
@@ -17,8 +17,7 @@ export class Position {
     this.column = col;
 
     // this.index = index;
-    // Object.defineProperty(this, "index", { enumerable: false, value: index });
-    indexes.set(this, index);
+    Object.defineProperty(this, "index", { enumerable: false, value: index });
```

Comment:
> In this current PR, I've kept it as an `enumerable: false` property, mainly to avoid having to generate all new fixtures, but I'd be happy to make it a normal property and do that too. 

Let's make it `enumerable`, even if it is a (one-time) annoying change that affects all our fixtures.